tests/unit/fixtures/__init__.py

```diff
@@ -13,4 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Fixtures that exclusively used in integration tests"""
+"""Fixtures that exclusively used in unit tests"""
+
+import os
+
+BASE_DIR = os.path.dirname(__file__)
```

Comment:
Pathlib?

Reply:
Oh yes sure, please go ahead and change it